auth/src/api/services/hashing-service.ts

```diff
@@ -1,12 +1,27 @@
 import crypto from 'crypto';
 
 import { HashingHandler } from '../interfaces/hashing-handler';
+import { Random } from '../../util/helper';
 
 export class HashingService extends HashingHandler {
     public hash(input: string): string {
+        if (!input) {
+            return '';
+        }
         return this.sha512(input);
     }
 
+    public isEquals(toHash: string, toCompare: string): boolean {
+        if (!toHash || !toCompare) {
+            return false;
+        }
+        if (toCompare.length === 152) {
+            return this.sha512(toHash, toCompare.slice(0, 64)) === toCompare;
+        } else {
+            return toHash === toCompare;
```

Comment:
`return false`. In which case it makes sense to compare a arbitrary string and a hashed version of it?